### ISSUE TITLE
Move new specs to Specifications section on drafts.csswg.org

### ISSUE THREAD

**Issue by SebastianZ** [w3c/csswg-drafts#6934]
Several of the upcoming specifications are currently listed under the "Other Documents" section on https://drafts.csswg.org. Those include at least CSS 2021, CSS Conditional 5, and CSS Values 5, maybe also some others there.

@plinss I guess you're the one that's maintaining that page, right?

Sebastian

**Comment by tabatkins** [comment]
Ah, that just means they're not actually added to Shepherd yet. I'll fix.

**Comment by tabatkins** [comment]
Done. (Tho the server had fallen and I just kicked it again, and now it's regenning a million things.)

**Comment by SebastianZ** [comment]
@tabatkins Thanks a lot!

**Comment by SebastianZ** [comment]
Reopening, because there's a bunch of new specs under "Other Documents" again, namely CSS 2022, CSS Cascade 6 (listed under css-scoping-2), CSS Scroll Snap 2, and CSS Easing 2.

@tabatkins If you show me what to do and I have the rights to make the changes, I'll do them myself next time.

Sebastian

**Comment by SebastianZ** [comment]
Thank you, @plinss!

Sebastian

**Comment by SebastianZ** [comment]
Re-opening for CSS Color 6.

Sebastian

**Comment by plinss** [comment]
Fixed, might take a few minutes to update

**Comment by SelenIT** [comment]
Probably [css-variables-2] should be in the Specs as well?

**Comment by plinss** [comment]
I presume there was a resolution to add it as an official draft? (I seem to remember that...)

What about [css-cascade-6]?

**Comment by SebastianZ** [comment]
> I presume there was a resolution to add it as an official draft? (I seem to remember that...)

There was a resolution in [discussion around custom units](https://github.com/w3c/csswg-drafts/issues/7379#issuecomment-1170229274).

> What about [css-cascade-6]?

css-cascade-6 is already in the specifications list. Though there's also css-scoping-2 currently redirecting to it, which seems to be wrong. @mirisuzanne, @tabatkins Is that reference intentional?

Sebastian

**Comment by plinss** [comment]
Right, I saw css-scoping-2 in the list and clicked it not realising it was redirecting to css-cascade-6. Not sure why it's in the 'Other Documents' section, should just be redirect on the server. I'll fix that and add css-variables-2.

**Comment by mirisuzanne** [comment]
`css-scoping-2` was briefly the home of the new `@scope` rule – but we resolved that `@scope` should move into `css-cascade-6` (resulting in the redirect), and the current `css-scoping-1` should be renamed to something more accurately related to shadow-context (which hasn't happened yet).

**Comment by plinss** [comment]
@mirisuzanne So the redirect to `css-cascade-6` is setup on the draft server (I do remember you asking for that now). The leftover `css-scoping-2` directory should probably just be deleted since no one can actually access it.

If/when you do rename css-scoping-1, please let me know and I can ensure that the old urls redirect to the new spec.

**Comment by SebastianZ** [comment]
Reopening, because there are several specs again that need to be moved, namely

* [CSS Anchor Positioning](https://drafts.csswg.org/css-anchor-1/)
* [CSS Images Module Level 5](https://drafts.csswg.org/css-images-5/)
* [CSS Linked Parameters](https://drafts.csswg.org/css-link-params/)
* [Web Animations CSS Integration](https://drafts.csswg.org/web-animations-css-integration/) (not sure what should happen with that one)

Sebastian

**Comment by plinss** [comment]
Moved the first 3, and moved css-link-params to css-link-params-1 in the process (there are a few other drafts that need level numbers too, but I didn't mess with those).

I don't know what to do with Web Animations CSS Integration either so leaving tha alone for now.

**Comment by SebastianZ** [comment]
Reopening for CSS 2023.

Sebastian

**Comment by plinss** [comment]
Added to the spec db, should move on the home page shortly.

**Comment by BenjaminAster** [comment]
@plinss [css-position-4](https://drafts.csswg.org/css-position-4/) should be added as well.

**Comment by SebastianZ** [comment]
Reopening again for fixing CSS View Transitions 2 (currently incorrectly named "CSS View Transitions 1") and Scroll-driven Animations 1 (currently still called "Scroll-linked Animations 1").
Asked @tabatkins back in August to fix that, though it seems it got missed.

Also, CSS 2024 can already be moved under "css-snapshot".

Sebastian

**Comment by SebastianZ** [comment]
@tabatkins @plinss Quick Ping. Please see my last comment!

Also, there are some specs missing their level:

* CSS Color Adjust ⇒ CSS Color Adjust 1
* CSS Scrollbars ⇒ CSS Scrollbars 1
* CSS Shadow Parts ⇒ CSS Shadow Parts 1
* CSS Viewport ⇒ CSS Viewport 1

Sebastian

**Comment by plinss** [comment]
Updated

**Comment by SebastianZ** [comment]
@plinss There are again three specs that need to be moved:

* [CSS Mixins 1](https://drafts.csswg.org/css-mixins-1/)
* [Selectors 5](https://drafts.csswg.org/selectors-5/)
* [CSS Overflow 5](https://drafts.csswg.org/css-overflow-5/)

Sebastian

**Comment by svgeesus** [comment]
@plinss if adding new specs is relatively simple, perhaps some additional logins would enable us the share some of that maintenance load?

**Comment by svgeesus** [comment]
Closing because the new index of drafts made by @sideshowbarker fixes all these long-running spec maintenance issues.

**Comment by frivoal** [comment]
@svgeesus : true literally, but https://drafts.fxtf.org/ and https://drafts.css-houdini.org/ are still on the old system. Should a new issue be opened about them?

**Comment by svgeesus** [comment]
https://drafts.fxtf.org/ has the index of all the old fxtf specs, but no new ones will be added, so it will not go out of date; and the links on individual specs redirect to drafts.csswg.org

(Houdini has not yet been moved over, but will behave the same once it has)

**Comment by frivoal** [comment]
> https://drafts.fxtf.org/ has the index of all the old fxtf specs, but no new ones will be added, so it will not go out of date;

But as I understand it, that index is hosted on a server which goes poof 3 days from now. It will not go out of date, but out of existence.

**Comment by emilio** [comment]
Can we "just" merge fxtf drafts and css-houdini drafts into csswg-drafts? (and set up redirects as needed?)

**Comment by svgeesus** [comment]
fxtf drafts are already merged (the specs, but not yet the issues). Houdini drafts will be merged too, that is on me to take the time to do.

So this is purely about not breaking links to the old sites.
